src/main/rendering/ImageWriter.java

```diff
@@ -1,21 +1,39 @@
 package rendering;
 
+import java.awt.image.BufferedImage;
+import java.awt.image.DataBufferInt;
+import java.io.File;
+import java.io.IOException;
+import javax.imageio.ImageIO;
 import primitives.Colour;
 
 /**
  * This class is responsible for writing pixels to an image file.
  *
- * @author
- * @author
  */
 public class ImageWriter {
+
+	/**  
+	 * The path where the image will be stored
+	 */
+	public static final String PATH = "images/";
+	public static final String EXT = ".png";
```

Comment:
Shouldn't we take this stuff as arguments? That way the user could specify any output in any folder with any file extension?

The argument `filename` should really be called `filepath` and should contain all this stuff.

Reply:
Could be. Instructions said to put it in images/
But that could be for just this exercise.

Reply:
Since when do we follow orders?